### ISSUE TITLE
Setting about the data_unit in the config.yaml for the evaluation in the paper

### ISSUE THREAD

**Issue by Carbord** [ybhbingo/MMFi_dataset#1]
Hello, it is an impressive work. After reading the paper and running the codes, I have some questions.
Q1: As for the pose estimation, do you use the single frame data as input and then output the single-frame pose, or adopt the sequence-in-sequence-output？ By the way, is it neccesary to divide the 297-frame sequence into small segments for evaluation?

Q2: While visualizing the GT 3d pose, mmwave point cloud and lidar point cloud, it seems that their x-y-z directions are different. Does this means that such directions of the input point cloud need to be rearranged before being fed into the learning model?

Sincerely looking forward to your reply. Thanks!

**Comment by ybhbingo** [comment]
Hi, 

1. If I am understanding correctly, you are wondering how the ground truth poses are generated? We optimize the poses of a sequence according to the loss function mentioned in the paper. 
I think small segments evaluation just depends on your task. If you just want evaluation for single frame or the whole 297-frame sequence, you can use the frame unit option in configuration file ("frame" or "sequence"). For example, if you specify the data unit as "frame", you would get a batch of random samples (each sample refers to a single frame, e.g., 1st frame may come from A01 of S02, while the 2nd frame may come from A10 of S27.). If "sequence" is specified, each sample from dataloader is a 297-frame sequence. But if you really need partially segmented sequence, you would customize the codes according to your requirements. 

2. This is a good question. It depends on your model. In our paper, the model we utilized is of point transformer structure which is invariant to permutation of the input elements. Actually, the parameters of model could be fitted automatically no matter which coordinate order the input data contains, and the only difference is the latent meaning of parameters. So, you can either rearrange the coordinates before feeding into a network model or just directly use the existing coordinates. By the way, we may add the coordinate transformation funtion for mmwave and lidar points into the mmfi.py later. 

You are welcome!
